compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -4717,6 +4722,17 @@ private void checkTypesForRecords(BLangWaitForAllExpr waitExpr, AnalyzerData dat
         }
     }
 
+    private boolean checkBinaryBitwiseOperations(BLangExpression valueExpr) {
```

Comment:
```suggestion
    private boolean isBinaryBitwiseOperatorExpr(BLangExpression valueExpr) {
```
Shall we change the method signature in this way since it returns a boolean.

Reply:
Updated. 3fca4a007e04efc7f97e82ca8c73f38adca29def